### ISSUE TITLE
Forbid duplicate `except` blocks

### ISSUE THREAD

**Issue by sobolevn** [wemake-services/wemake-python-styleguide#363]
# Rule request

## Thesis

We need to raise a violation when we see cases like:

```python
try:
    1 / 0
except ZeroDivisionError:
    pass
except ZeroDivisionError:
    pass
```

Or

```python
try:
    1 / 0
except ZeroDivisionError:
    pass
except (IndexError, ZeroDivisionError):
    pass
```

## Reasoning

This is unclear and will lead to errors in the future.
